packages/twenty-front/src/modules/activities/inline-cell/components/ActivityTargetsInlineCell.tsx

```diff
@@ -39,13 +39,14 @@ export const ActivityTargetsInlineCell = ({
           scope: ActivityEditorHotkeyScope.ActivityTargets,
         }}
         IconLabel={IconArrowUpRight}
+        showLabel={true}
         editModeContent={
           <ActivityTargetInlineCellEditMode
             activity={activity}
             activityTargetObjectRecords={activityTargetObjectRecords}
           />
         }
-        label="Relations"
+        label="Relations:"
```

Comment:
should we add the semi column? It looks like we don't have it for other fields

Reply:
cc @Bonapara since it was like that on his design

Reply:
Yes well noticed, we can get rid of the semicolon!